src/OpenTelemetry/Batch.cs

```diff
@@ -50,9 +53,13 @@ public Batch(T[] items, int count)
             this.Count = this.targetCount = count;
         }
 
-        internal Batch(T item)
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Batch{T}"/> struct.
+        /// </summary>
+        /// <param name="item">The item to store in the batch.</param>
+        public Batch(T item)
```

Comment:
I decided to make this public. So that [ReentrantExportProcessor<T>](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/827eb256890889aeb72d0d641beb92ebd9819b94/src/OpenTelemetry.Exporter.Geneva/ReentrantExportProcessor.cs#L28) can be updated to participate in pooling without needing reflection.

Reply:
FYI - there is a related spec PR https://github.com/open-telemetry/opentelemetry-specification/pull/2452 which I'm trying to help out.

Reply:
OK it sounds like we are working to pull `ReentrantExportProcessor<T>` into the main repo so I reverted this change.